app/assets/javascripts/pageflow/ui/views/inputs/text_area_input_view.js

```diff
@@ -68,6 +68,9 @@ pageflow.TextAreaInputView = Backbone.Marionette.ItemView.extend({
           u: {unwrap: this.options.disableRichtext ? 1 : 0},
           b: {unwrap: this.options.disableRichtext ? 1 : 0},
           i: {unwrap: this.options.disableRichtext ? 1 : 0},
+          ol: {unwrap: this.options.disableRichtext ? 1 : 0},
+          ul: {unwrap: this.options.disableRichtext ? 1 : 0},
+          li: {unwrap: this.options.disableRichtext ? 1 : 0},
```

Comment:
Shouldn't this depend on `enableLists` instead?

Reply:
`enableLists` controls whether or not the list-buttons are shown. `disbaleRichtext` is an independent option which causes the tags to get unwrapped/converted to plain text

Reply:
Yes. But `u`/`b`/`i` tags are unwrapped when `disableRichtext` is set to ensure those get stripped when pasting text from the clipboard. Wouldn't we want to strip out lists in copied texts as well if `enableLists` is false even if rich text is allowed?